baggers.js

```diff
@@ -5520,17 +5520,17 @@ var data = {
           lang: ['fr'],
         },
         {
-          title: 'WebRTC : de la visio dans mon navigateur sans plugin ! ',
+          title: "J'ai organisé un atelier pour les enfants",
           abstract:
-            "Avez déjà rêvé de faire une visio avec votre grand mère, elle sur son smartphone et vous installés dans votre canapé avec votre ordinateur portable. Le problème : elle n'a pas Skype, pas de compte Google et elle ne connait pas Facetime ? Et si votre navigateur était la solution  ! Vous pourrez même lui partager les photos de votre petite dernière.Vous l'aurez compris, l'objectif de cette session est de présenter ce qu'est le WebRTC et de comprendre ce qui ce cache derrière.",
-          tags: ['Mobile', 'Android', 'NFC', 'HTML5', 'WebRTC', 'Sensors'],
+            "Les Devoxx4Kids, vous connaissez ? Mais si vous savez, ces ateliers pour faire découvrir la programmation aux plus petits ? Je vais vous faire un petit retour d'expérience sur les choix que nous avons fait à Nantes et pourquoi nous les avons fait. Comment avons nous choisit le programme ? Combien de temps doit durer les ateliers ? .... \n Enfin je ferais une démo de scratch pour que vous puissiez expérimenter cet outils avec vos enfants",
+          tags: ['Enfants', 'Découverte', 'Devoxx4Kids'],
           lang: ['fr'],
         },
         {
-          title: "J'ai organisé un atelier pour les enfants",
+          title: "JS in CSS: Yes we can 🤔",
           abstract:
-            "Les Devoxx4Kids, vous connaissez ? Mais si vous savez, ces ateliers pour faire découvrir la programmation aux plus petits ? Je vais vous faire un petit retour d'expérience sur les choix que nous avons fait à Nantes et pourquoi nous les avons fait. Comment avons nous choisit le programme ? Combien de temps doit durer les ateliers ? .... \n Enfin je ferais une démo de scratch pour que vous puissiez expérimenter cet outils avec vos enfants",
-          tags: ['Enfants', 'Découverte', 'Devoxx4Kids'],
+            "Alors que tout le monde parle de CSS in JS et si on regardait comment faire du JS in CSS ? Car oui c’est possible ! Est ce une bonne idée ? Je n’en suis pas convaincu mais pourquoi ne viendrez pas vous détendre sur cette session à la fois technique et fun.\n“Because we can” n’est ce pas ce qui nous motive dans certains de nos développements 😉",
```

Comment:
```suggestion
            "Alors que tout le monde parle de CSS in JS et si on regardait comment faire du JS in CSS ? Car oui c’est possible ! Est-ce une bonne idée ? Je n’en suis pas convaincu mais pourquoi ne viendrez pas vous détendre sur cette session à la fois technique et fun.\n“Because we can” n’est ce pas ce qui nous motive dans certains de nos développements 😉",
```